app/src/lib/git/log.ts

```diff
@@ -29,6 +29,9 @@ function mapStatus(
   if (status === 'A') {
     return { kind: AppFileStatusKind.New }
   } // added
+  if (status === '?') {
```

Comment:
i'm not sure if this is necessary since this is a `git log`, but i'm including it for completeness